docs/en/ingest-management/elastic-agent/configuration/autodiscovery/kubernetes-conditions-autodiscover.asciidoc

```diff
@@ -0,0 +1,318 @@
+[[conditions-based-autodiscover]]
+= Conditions based autodiscover
+
+You can define autodiscover conditions in each input to allow {agent} to automatically identify Pods and start monitoring them using predefined integrations. You can find more about these inputs and their configurations <<elastic-agent-input-configuration,here>>.
```

Comment:
I corrected the wording. I also made same changes to the inputs config file to make it more clear. It's not complete, but at least for now, it has some definitions for the fields. Do you think it makes sense to point to that page, or should that be removed all together?

Reply:
I pointed to the System module in one of the definitions. I am not sure if it was the right thing to do, but I wanted to give the a user a direction for finding the metricsets

Reply:
>Do you think it makes sense to point to that page, or should that be removed all together?

Actually I think you should point to that page because it's better than nothing, and eventually users will be able to find content that's more meaningful there.

> I pointed to the System module in one of the definitions. I am not sure if it was the right thing to do, but I wanted to give the a user a direction for finding the metricsets

I think that's the best you can do right now. Might be worth adding a sentence that explains why you're pointing to the Metricbeat docs. (Something about how the input configuration settings for standalone agents are not yet documented, but inputs support the same settings available in Metricbeat as metricsets.)